internal/commands/app/create_inputs.go

```diff
@@ -108,13 +108,16 @@ func (i *createInputs) resolveLocalPath(ui terminal.UI, wd string) (string, erro
 	if !fi.Mode().IsDir() {
 		return fullPath, nil
 	}
-	_, appOK, err := local.FindApp(fullPath)
+	_, _, err = local.FindApp(fullPath)
 	if err != nil {
 		return "", err
 	}
-	if appOK {
-		return "", errProjectExists{fullPath}
+
+	if ui.AutoConfirm() {
+		fullPath = path.Join(wd, i.LocalPath+"-1")
```

Comment:
it would be nice if we could somehow verify the `-1` doesn't already exist too, and potentially keep incrementing up until we find a directory that _is_ available on disk.

also, this unique path could then be set to `Default` on the input prompt below